4-esqueleto/README.md

```diff
@@ -1,3 +1,76 @@
 # Plano de esqueleto
 
-## Referencias
\ No newline at end of file
+## Wireframes de las páginas de Aupa
+
+### Introducción (Lisandro)
+
+En este documento se ha de desarrollar el plano esqueleto, para ello, primero se detectan y definen las páginas del sitio web Aupa, luego se realiza un wireframe por cada una y por último abordando cada una de las páginas con la perspectiva de los tres tipos de diseños:
+
+- Diseño de la interfaz
+- Diseño de la navegación
+- Diseño de la información
+
+A continuación se muestra el resultado de este proceso.
+
+### Página principal (Angie)
+
+### Buscador (Estelle)
+
+![Buscador](buscador.png)
+
+### Contacto (Estelle)
+
+![Contacto](contactos.png)
+
+### Foro 
+![IMG1](/4-esqueleto/Wireframes.jpeg)
+
+En la segunda imagen, podemos ver la pagina principal de la pestaña “foro”. 
+Para la interfaz, será una interfaz muy simple y para la navegación, será muy fácil, podremos ver los sujetos del foro, el numero de vistas y de posts serán visibles. 
+En la primera, es la pagina que los usuarios podrán ver cuando van a hacer click en un sujeto. La navegación va a ser muy simple como podremos ver en la imagen.
+
+### Páginas de similar diseño (Lisandro)
```

Comment:
No sé si tiene sentido una sección con este nombre para la presentación del documento. Aunque lo hayáis usado para distribuir el trabajo, dadle una vuelta a las secciones del documento y sus nombres para la entrega final.

Reply:
Voy a mejorar el titulo para que quede mejor explicado. Pero entendemos esta sección como un medio para abordar el esquema de Wireframes que tendrán los artículos que se escriban en las distintas pestañas del sitio web y que tendrán un tratamiento semejante.
Al no producir contenido nuevo, sino más bien traducir le contenido existente en artículos útiles para el publico objetivo, los esquemas serán idénticos en términos de navegación y estética.

Reply:
Chicas yo voy a introducir los cambios aquí.